t/v27_features.py

```diff
@@ -20,3 +20,92 @@
 assert(result['uri'] == '/issue_120/{}'), "expected result"
 print ('[+] OK')
 
+preset_method_location = BASE_URL + '/skipping_test'
+body = [
+  {
+    "skipping_map_at_the_beginning": {
+      "ttl": 123
+    },
+    "id": 1,
+    "method": "skipping_test",
+    "params": []
+  },
```

Comment:
An order of keys in a dictionary is arbitrary in Python. Maybe we can use `OrderedDict` from `collections` to ensure certain order in the resulting JSON (but, please, verify it).

Reply:
Done.

Reply:
It seems, it should work: the `OrderedDict` instances are passed to `json.dumps()`, which preserves order in the case. Okay.